communitymanager/src/main/java/org/pesmypetcare/communitymanager/managers/ForumManagerClient.java

```diff
@@ -161,6 +162,7 @@ public void postMessage(String token, String parentGroup, String forumName, Mess
      */
     public void deleteMessage(String token, String parentGroup, String forumName, String creatorName, String date)
         throws MyPetCareException {
+        date = DateTime.convertLocalToUTCString(date);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assignment of parameter 'date' is not allowed.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5639007)